war/src/main/webapp/help/system-config/master-slave/usage.html

```diff
@@ -19,11 +20,11 @@
       that project is restricted to certain nodes using a label expression,
       and that expression matches this node&apos;s name and/or labels.
 
-      This allows a slave to be reserved for certain kinds of jobs.
+      This allows a node to be reserved for certain kinds of jobs.
```

Comment:
@jtnord Yes, I wish we didn't have to have this inconsistency :disappointed: 

I kept the wording here as "node" because the launcher mode is currently called "Utilize this **node** as much as possible", and the rest of the text talks about "nodes" — even although we're on the *agent* configuration page, and all the other inline documentation here talks about "agents".

I opened [JENKINS-33007](https://issues.jenkins-ci.org/browse/JENKINS-33007) recently; I believe all instances of "node" should be replaced with "agent".  Just because internally everything is called a `Node`, does not mean that users should be subjected to this terminology.  Even before the "slave" -> "agent" rename, this could have been better, but it makes more sense now that slaves have a more descriptive name.

The Jenkins master server has two roles: it handles configuration, queue management etc.  It also can optionally act as an *agent*, executing builds, much like any other SSH or JNLP or whatever agent.

So in 2.0 we currently end up with this delightful mixture of terms:
![agent-node-mixture](https://cloud.githubusercontent.com/assets/138615/13638057/d8da961a-e60a-11e5-9d34-dddd4a500fdb.png)

I know @daniel-beck wants to keep "node", but I haven't been able to understand why.  I really think JENKINS-33007 would be a beneficial change for consistency and clarity, and therefore end users, but for this PR I'm sticking to what fits with the immediate context (i.e. Utilize this node).